www/css/index.css

```diff
@@ -308,4 +309,15 @@ max width and height?
   position: absolute;
   top:0;
   /*border: 1px solid red;*/
+}
+
+/* Answer Collect Panel Styles */
+.collectAnswerPanel {
+  position: fixed;
+  top: 2.5%;
+  bottom: 2.5%;
+  left: 5%;
+  right: 5%;
+  background-color: rgba(255,255,255,.8);
+  z-index: 100;
```

Comment:
@dooderlani - I totally just got some super basic markup and styling for this Answer Collecting Panel (visual feedback for when it's a correct answer) so feel free to set this up and style it however you want! Same with the verbiage in the markup!